### ISSUE TITLE
[Feature request] New command for creating file relative to current, without prompting for directory

### ISSUE THREAD

**Issue by ryaninvents** [patbenatar/vscode-advanced-new-file#110]
I use this extension constantly in large projects, and I rarely select a location other than "current file". However, I must still wait for the list of directories to load. Would there be interest in a new command `extension.advancedNewFileHere` that always creates the new file(s) relative to the open file? This way, the two commands could be bound to separate shortcuts and I could choose which to use based on what I'm doing.

I'd be happy to submit a PR for this; just wanted to gauge interest first before committing to anything.

**Comment by ryaninvents** [comment]
Possibly related to #58

**Comment by patbenatar** [comment]
@ryaninvents Agreed! That'd be a great feature. I don't have much time to work on this plugin these days -- although I do still use it all the time. If you're interested in PRing the feature, I'd welcome the contribution with open arms 😃
